crates/sui-graphql-rpc/src/test_infra/example.move

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This file shows we can run normal Move transactional tests along with custom simulator commands.
This will serve as the data source for the test runs